32-fall-detection-and-panic-button/README.md

```diff
@@ -0,0 +1,122 @@
+* [1 push button](https://www.sparkfun.com/products/14460)
+* [SparkFun Micro Triple Axis Accelerometer Breakout - BMA400 (Qwiic)](https://www.sparkfun.com/products/21207)
+* [Qwiic Cable - Breadboard Jumper (4-pin)](https://www.sparkfun.com/products/14425)
+* 2 USB A to micro USB cables
+* Breadboard
+* Jumper wires
+* Solder flux
+* Soldering iron
+* Tape
+
+## Notehub Setup
+
+Sign up for a free account on [notehub.io](https://notehub.io) and [create a new project](https://dev.blues.io/quickstart/notecard-quickstart/notecard-and-notecarrier-a/#set-up-notehub).
+
+## Sparrow Setup
+
+### Quickstart
+
+Follow the [Sparrow Quickstart](https://dev.blues.io/quickstart/sparrow-quickstart/) to get your Sparrow reference node paired with the gateway and associated with the Notehub project you just created. Note also that we'll only need one reference node for this project, so you don't need to pair both nodes that came with the dev kit. Make sure that you use the ProductUID generated in [Notehub Setup](#notehub-setup) when it comes time to issue the `hub.set` command in the quickstart.
+
+After you've completed the quickstart, leave the Notecarrier and essentials board powered and connected. These two devices will act as our gateway to Notehub, and we won't need to touch them again. The rest of this guide will focus on the reference node.
+
+### Hardware
+
+There are two primary pieces of hardware: the button and the accelerometer.
+
+The button will be wired to a GPIO pin on the reference node. This pin uses an internal pull-up so that the it reads a digital 1 when the button isn't pushed. When the button is pushed, it connects the corresponding GPIO pin to GND so that it reads a digital 0. Using the breadboard, button, and jumper wires, wire up the hardware as shown in the diagram below, connecting one side of the button to A2 and the other to the minus (-) rail of the breadboard. Make sure to then connect the minus rail to the reference node's GND port.
+
+![breadboard button wiring](assets/breadboard_button.jpg "Breadboard Button Wiring")
+
+Now, when the button is pressed, it will connect A2 to GND.
+
+The accelerometer board has a hole on it labeled INT1. This is the interrupt line that our app will use to detect a fall event, so we need to connect it to the reference node. To do that, grab a male to male jumper wire that will fit in the hole. Stick one end of the wire as far as it'll go through the hole, so that when it's soldered in, we can plug it into the breadboard. Using your soldering iron and flux, solder the jumper into place. Seat the accelerometer onto the breadboard using the exposed end of the wire sticking out under the accelerometer board. Then, connect the other end of the wire to pin A1 on the reference node.  The accelerometer also has a Qwiic port that needs to be wired up to the Sparrow reference node. Plug the Qwiic breadboard jumper cable into this port, and then on the other end make the following connections:
```

Comment:
I'm wondering about the tone here - "grab" and "stick". Those words feel (to me at least) a bit slapdash. 

```suggestion
The accelerometer board has a hole on it labeled INT1. This is the interrupt line that our app will use to detect a fall event, so we need to connect it to the reference node. To do that: 

1. Take a male-to-male jumper wire that fits the hole.
2. Push one end of the wire as far as it'll go through the hole. This is so that when it's soldered in, we can plug it into the breadboard. 
3. Using your soldering iron and flux, solder the jumper into place. 
4. Seat the accelerometer onto the breadboard using the exposed end of the wire sticking out under the accelerometer board.
5. Connect the other end of the wire to pin A1 on the reference node.  

The accelerometer also has a Qwiic port that needs to be wired up to the Sparrow reference node.  Plug the Qwiic breadboard jumper cable into this port, and then on the other end make the following connections:
```

Reply:
Regarding the size of the jumper wire, do the usual Dupont jumpers work or did you have to use something else?  I It would be good if we can advise on which wire to use, or the gauge of the wire if the standard ones aren't a good fit.

Reply:
I have a bunch of square and round jumper wires. I'm trying to find the terminology for each, but I'm drawing a blank on Google.

Basically, my round ones look something like this: https://www.amazon.com/Breadboard-Jumper-Wire-75pcs-pack/dp/B0040DEI9M

And the square ones look like this: https://www.amazon.com/SUNKEE-Dupont-Color-Jumper-2-54mm/dp/B00AX3PHV6

The former fit into the BMA400 breakout, while the latter don't. I'll look through the Sparkfun docs for the breakout and see if I can be more specific about the size of jumper needed.